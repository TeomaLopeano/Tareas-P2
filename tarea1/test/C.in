insertarAlInicio 23 23.0
cantidadEnCadena
infoCadena 23
insertarAlInicio 11 11.1
insertarAlInicio 21 21.0
insertarAlInicio 8 8.0
insertarAlInicio 13 13.0
imprimirCadena
estaEnCadena 11
estaEnCadena 12
estaEnCadena 13
infoCadena 11
infoCadena 13
Fin