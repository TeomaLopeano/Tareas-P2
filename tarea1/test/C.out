1>Se insertó (23,23.00) al inicio de la cadena.
2>La cantidad es 1.
3>El elemento es (23,23.00)
4>Se insertó (11,11.10) al inicio de la cadena.
5>Se insertó (21,21.00) al inicio de la cadena.
6>Se insertó (8,8.00) al inicio de la cadena.
7>Se insertó (13,13.00) al inicio de la cadena.
8>(13,13.00)(8,8.00)(21,21.00)(11,11.10)(23,23.00)
9>11 está en la cadena.
10>12 no está en la cadena.
11>13 está en la cadena.
12>El elemento es (11,11.10)
13>El elemento es (13,13.00)
14>Fin.
